app/controllers/api/v2/reports_controller.rb

```diff
@@ -59,6 +59,12 @@ def last
         @report = Report.authorized(:view_reports).includes(:logs => [:message, :source]).find(max_id)
```

Comment:
Lots of references to Report in this controller to change to ConfigReport, and probably override resource_class.